addons/crm_livechat/models/discuss_channel.py

```diff
@@ -45,6 +60,7 @@ def _convert_visitor_to_lead(self, partner, key):
 
         utm_source = self.env.ref('crm_livechat.utm_source_livechat', raise_if_not_found=False)
         return self.env['crm.lead'].create({
+            "channel_id": self.id,
```

Comment:
It would be nice to adapt at least one existing to ensure this is properly written.